app/cron.js

```diff
@@ -69,7 +76,7 @@ class CronManager {
           const cronJobSetPromises = [];
           for (const job of jobs) {
             cronJobSetPromises.push(
-              this._setCronJob(tenant, job.jobId, job.spec)
+              this.setCronJob(tenant, job.jobId, job.spec)
```

Comment:
Split this code into two methods if possible (following the Single Responsibility Principle, google it kkk)

Reply:
Vou de português aqui:
Bem, esse é um serviço experimental e este é só um dos problemas a serem tratados, está prometido um ticket especifico para resolver essa e outras questões, como testes unitários.